user-guide/modules/ROOT/pages/libraries-that-work-together.adoc

```diff
@@ -0,0 +1,32 @@
+
+* boost:graph[] and boost:property_map[] : Boost.Graph is a library for working with graphs, and Boost.PropertyMap provides a generalized interface to access property maps. These two libraries can be used together to perform computations on graphs, such as shortest path calculations, maximum flow, and connected component detection.
+
+* boost:mpl[] and boost:fusion[] : Boost.MPL provides a framework for metaprogramming, allowing for compile-time computations, and Boost.Fusion provides a framework for working with tuples, providing functionality to treat tuples like compile-time lists. These libraries are useful together for template metaprogramming tasks.
+
+* boost:spirit[] and boost:phoenix[] : Boost.Spirit is a library for creating parsers and output generation, and Boost.Phoenix is a library for functional programming. They are often used together for tasks like parsing complex data formats.
+
+* boost:bimap[] and boost:multi_index[] : Boost.Bimap is a library for bidirectional maps, and Boost.MultiIndex supports data structures allowing for access and retrieval based on multiple indices. These libraries work well together for data storage and retrieval tasks where complex access patterns are needed.
+
+* boost:geometry[] and boost:range[] : Boost.Geometry provides classes and functions for geometry computations, and Boost.Range provides a new concept to the Standard Library's iterator pair. They can be used together for computations on sets of geometric objects.
+
+* boost:math[] and boost:units[] : Boost.Math provides functions for mathematical computations, and Boost.Units provides classes and functions for working with different systems of units. They work well together for scientific calculations where unit conversion is important.
+
+== Working Well in Multiple Combinations
+
+[disc]
+* boost:serialization[] and boost:filesystem[], boost:multi_array[], or or boost:ptr_container[] : Boost.Serialization is used for saving and loading data, which is a common task in many types of software, including games. After you've serialized your data with Boost.Serialization, you'll often want to write it to disk. Boost.Filesystem provides a portable and easy-to-use interface for file and directory operations. Boost.MultiArray provides a generic N-dimensional array concept definition and common implementations of that interface. It's great for storing game data that naturally fits into a grid or higher-dimensional structure. These arrays can be easily serialized with Boost.Serialization. Boost.Pointer_Container provides containers for storing heap-allocated or cloned objects, which can be serialized with Boost.Serialization. This is useful when you need polymorphic serialization.
```

Comment:
> Boost.Serialization. Boost.Pointer_Container

No links

Reply:
Linked on first use:  boost:serialization[] and boost:filesystem[], boost:multi_array[], or or boost:ptr_container[] ?

Could make all references links.

Reply:
Yes. The other names without links look bad.